src/pages/questionPage.js

```diff
@@ -28,33 +28,58 @@ export const initQuestionPage = () => {
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
 
-  for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
-    const answerElement = createAnswerElement(key, answerText);
+  answersListElement.innerHTML = '';
+  const answers = currentQuestion.answers;
+
+  // Function to handle answer selection
+  const selectAnswer = (questionIndex, selectedOption, chosenItem) => {
```

Comment:
Function `selectAnswer` needs to be outside of `initQuestionPage` function. Please put it outside of that scope and make it as an independent function like Ahmad and myself did. 
Also, remove "chosenItem" as parameter because you're not using here.